src/pages/library/Showcase.js

```diff
@@ -36,9 +44,46 @@ const sidebarSections = [{
   labels: ['HCA', 'AnVIL', 'BRAIN Initiative', 'BioData Catalyst', 'NCI']
 }]
 
+
+const WorkspaceCard = variant => workspace => {
```

Comment:
Take a look at the updated `WorkspaceCard` - you should no longer need to call it with parens